sdlf-utils/ingestion-examples/preprocessing/fargate-unzip/Dockerfile/publish_docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+#set -nx
+# Execution Instructions
+# must be executed in the same directory of Dockerfile and .sh
+# Get your Account ID
+dockerContainerName="unzip-shell"
+awsRegion="us-east-1"
```

Comment:
Is the region value overridden when not deploying to `us-east-1`?

Reply:
This script is part of the preparations steps that you should execute before deploying the actual artifact. This is why I did not add any parameter for this value and you should modify it directly in the shell script. So, short answer is yes, you should override this when not deploying to us-east-1.